builders/manage/dapp/manage-tokens.md

```diff
@@ -0,0 +1,131 @@
+
+1. Select the **registrar** storage module
+2. Select **registrarDeposit**
+3. Insert your appchain id
+4. Press **+** icon
+
+You'll see the registration account at the bottom.
+
+![Locating your registration account](/images/builders/manage/dapp/token-management/token-management-1.webp)
+
+### Retrieving the Sudo Account {: #retrieving-sudo-account }
+
+If you're unsure what your Tanssi appchain's Sudo account is, you can find it in your [Tanssi Dashboard](https://apps.tanssi.network){target=\_blank} underneath the **Properties** section.
+
+![Locating your Sudo address on apps.tanssi.network](/images/builders/manage/dapp/token-management/token-management-2.webp)
+
+!!! warning
+    You should always protect your Sudo account key with the utmost security precautions, as it grants privileged access to your Tanssi appchain.
+
+## Accessing the Token Management Panel {: #accesing-token-management-panel }
+
+The Tanssi dApp implements a smooth interface, allowing the appchain governor to access and execute privileged functions. To do so, head to the [Tanssi dApp](https://apps.tanssi.network/){target=\_blank}, click on the **Manage** button, and then, again click on the **Token Management** button.
+
+![Accessing the token management panel](/images/builders/manage/dapp/token-management/token-management-3.webp)
+
+Now you have direct access to the actions presented in the introduction:
+
+1. **Mint Tokens**
+2. **Update Balances**
+3. **Transfer Tokens**
+4. **Configure Gas Dynamics** 
+
+![The token management panel](/images/builders/manage/dapp/token-management/token-management-4.webp)
+
+!!! note
+    If you don't see a card with your appchain's details, make sure to comply with the [prerequisites](#checking-prerequisites).
+
+### Minting Tokens {: #minting-tokens }
+
+In the **Token Management** panel, click on the **Mint Tokens** button, and then:
+
+1. Insert the address that will hold the newly minted tokens
+2. Insert the amount of tokens to mint
+3. Click on **Mint**
+
+![Mint Tokens](/images/builders/manage/dapp/token-management/token-management-5.webp)
+
+You'll be asked to sign the transaction with the appchain's governor account. Once the transaction has gone through, the destination account's balance will have been increased by the desired amount.
+
+!!! note
+    The destination address must be Ethereum type if the chain is EVM-compatible and a Substrate type otherwise.
+
+### Updating Balances {: #updating-balances }
+
```

Comment:
This is another way to mint tokens? Or is this simply sending tokens from the existing governor balance?

Reply:
It's just another way to mint or burn tokens (you can also decrease a balance)